components/uikit/Input/dumb/DateInputDumb.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { useState } from "react";
+import DateTimePickerModal from "react-native-modal-datetime-picker";
+import {
+  Box,
+  Input,
+  InputField,
+  InputSlot,
+  FormControl,
+  FormControlError,
+  FormControlErrorText,
+  InputIcon,
+  FormControlErrorIcon,
+  AlertCircleIcon,
+} from "@gluestack-ui/themed";
+import { CalendarMonth } from "@/components/icons/calendarMonth";
+
+export type DateInputProps = {
+  error?: string;
+  isRequired?: boolean;
+  value?: Date;
+  onChange: (date: Date) => void;
+};
+
+export const DateInput = ({
+  error,
+  isRequired,
+  value,
+  onChange,
+}: DateInputProps) => {
+  const [isDatePickerVisible, setDatePickerVisibility] = useState(false);
+
+  const showDatePicker = () => {
+    setDatePickerVisibility(true);
+  };
+
+  const hideDatePicker = () => {
+    setDatePickerVisibility(false);
+  };
+
+  const handleConfirm = (date: Date) => {
+    onChange(date);
+    hideDatePicker();
+  };
+
+  const dateString = value?.toLocaleDateString();
+
+  const isInvalid = error ? true : false;
+
+  return (
+    <Box>
+      <FormControl
+        isRequired={isRequired}
+        isReadOnly={true}
+        isInvalid={isInvalid}
+      >
+        <Input
+          px="$2"
+          h={58}
+          borderRadius="$xl"
+          borderWidth="$1"
+          borderColor="$primaryDark"
+          $focus-borderWidth={1.5}
+        >
+          <InputField
+            type="text"
+            placeholder="Date de naissance"
```

Comment:
Don't hard code this. Add placeholder prop to the component

Reply:
Okay but it will always be "Date de naissance" though...